app/(tabs)/shoppingList/index.tsx

```diff
@@ -239,13 +244,13 @@ export default function ToBeBoughtScreen() {
                 pointerEvents='none'
               />
             </View>}
-            data={members.map(m => {return {label: m.name, value: m.name}})}
+            data={members.map(m => {return {label: m, value: m}})}
             labelField="label"
             valueField="value"
             placeholder={'Selected: ' + selectedUsers.length}
-            value={selectedUsers.map(u => u.name)}
+            value={selectedUsers.map(u => u)}
```

Comment:
Can't you just write `value={selectedUsers}`? You are  just mapping the object to it self